Find a functionally equivalent code:android.os.Process.myUid:COMMENT
Method Modifier: public      static      final       
Comment:/**
 * Returns the identifier of this process's uid.  This is the kernel uid
 * that the process is running under, which is the identity of its
 * app-specific sandbox.  It is different from {@link #myUserHandle} in that
 * a uid identifies a specific app sandbox in a specific user.
 */

@@ -1,3 +1,3 @@
 {
-    return Libcore.os.getuid();
+    return Os.getuid();
 }

